libs/service-portal/social-insurance-maintenance/src/screens/PaymentPlan/PaymentPlan.tsx

```diff
@@ -45,6 +45,22 @@ const PaymentPlan = () => {
     })
   }, [selectedYear, getPaymentPlanQuery])
 
+  const totalPaymentsReceived =
+    data?.socialInsurancePaymentPlan?.paymentGroups
+      .filter((pg) => pg.type !== 'Frádráttur')
+      .reduce(
+        (total, current) => (total += current.totalYearCumulativeAmount),
+        0,
+      ) ?? 0
```

Comment:
Is fallback of `0` OK?

Reply:
stakeholders will have to make a decision this